transit_odp/publish/forms/seasonal_services.py

```diff
@@ -13,12 +13,18 @@
 from config.hosts import PUBLISH_HOST
 from transit_odp.common.constants import DEFAULT_ERROR_SUMMARY
 from transit_odp.organisation.models import Licence, SeasonalService
-from transit_odp.publish.forms.constants import CONTINUE_BUTTON, SAVE_BUTTON, SEPARATOR
+from transit_odp.publish.forms.constants import (
+    CONTINUE_BUTTON,
+    SAVE_BUTTON,
+    SEPARATOR,
+    get_table_page,
+)
 
 
-def cancel_seasonal_service(org_id):
+def cancel_seasonal_service(org_id: int, page: str) -> LinkButton:
```

Comment:
We dont need to return to a specific page here.
Just return to page 1
Its cancelling out of the add new wizard